daal4py/sklearn/monkeypatch/linear.py

```diff
@@ -157,3 +158,17 @@ def predict(self, X):
     else:
         X = check_array(X)
         return _daal4py_predict(self, X)
+
+
+class LinearRegression(LinearRegression_original):
+    __doc__ = LinearRegression_original.__doc__
+
+    def __init__(self, fit_intercept=True, normalize=False, copy_X=True,
+                 n_jobs=None):
+        super(LinearRegression, self).__init__(
+            fit_intercept=fit_intercept, normalize=normalize,
+            copy_X=copy_X, n_jobs=n_jobs)
+
+
+setattr(LinearRegression, 'fit', fit)
```

Comment:
Any particular reason why you do not define the functions inline?
The 'normal' way is to define class-methods inline.

Reply:
Less copy-pasting is the only reason.